jack-and-the-beans/buddies-backend
# Delete chat when activity is deleted

This firebase function is triggered when an activity is deleted. It deletes the activity's chat sub-collection. _Note: I have already deployed this and confirmed that it works in the wild_

To test:
1. Create a new activity, or find one of which you're the owner.
2. Find that activity in the Firestore console and navigate to its chat collection. It might be easiest to see if you select one of the chat documents.
3. Delete the activity
4. Within a few seconds, the chat data that you selected should be deleted.
